teoria/2.cono-cilindro.tex

```diff
@@ -0,0 +1,15 @@
+\begin{multicols}{2}
+	\textbf{Cono generalizzato}: \\
+	$\begin{cases}
+		\overrightarrow{VP}=t\cdot\overrightarrow{VP_{0}}& P_{0}\in \mathfrak{C} \\
```

Comment:
Posso consigliarti di inserire un breve paragrafo prima dei 3 sistemi che spieghi il significato di ogni "lettera" e, visto che adesso avanza tantissimo spazio, magari anche una spiegazione rapida del procedimento (es. per le quadriche di rotazione: si prende un punto generico sull'asse, si crea una sfera centrata in quel punto e con raggio uguale alla distanza dal centro al generico punto... ecc...)

Visto che lo spazio non manca affatto togliendo la sezione sulle forme quadratiche sentiti libero di togliere le abbreviazioni che ho fatto. In molti esercizi, specie quelli di Marco, ho dovuto commentare molte frasi, se ti sembrano utili puoi decommentare quelle righe.